src/ja/server/config.py

```diff
@@ -73,7 +73,7 @@ def from_dict(cls, property_dict: Dict[str, object]) -> "LoginConfig":
         host = cls._get_str_from_dict(property_dict=property_dict, key="host")
         port = cls._get_int_from_dict(property_dict=property_dict, key="port", mandatory=False)
         username = cls._get_str_from_dict(property_dict=property_dict, key="username")
-        password = cls._get_str_from_dict(property_dict=property_dict, key="password")
+        password = cls._get_str_from_dict(property_dict=property_dict, key="password", mandatory=False)
```

Comment:
When is password not needed?

Reply:
When I changed the default in the config from test123 to None.
The change to the mandatory keyword makes it so that the server doesn't immediately crash in that case.

Reply:
But without a password, it probably won't work at all? Or does it work?

Reply:
I forgot that we don't have unit tests for the actual server configuration files; it does not work.
I made passwords mandatory again.